application/org.openjdk.jmc.console.agent/pom.xml

```diff
@@ -44,7 +44,6 @@
 	</parent>
 
 	<artifactId>org.openjdk.jmc.console.agent</artifactId>
-	<version>8.1.0-SNAPSHOT</version>
```

Comment:
Is it ok to remove this version element?

Reply:
Yep. It shouldn't be there in the first place, rather inherited.